getTopic.py

```diff
@@ -0,0 +1,18 @@
+## getTopic
+
+def getTopic(text):
+    
+    getTopic_1 = countWords(text)
```

Comment:
Your variables should be named after what they mean. When another developer reads your code, they should be able to easily understand what the variable contains. For example instead of getTopic_1, you could name it wordCounts, etc.

Reply:
You can merge this PR after renaming your variables appropriately.